playpen/ansible/roles/dev/files/bashrc

```diff
@@ -19,44 +21,61 @@ source /usr/bin/virtualenvwrapper.sh
 
 alias phttp="http --verify no --cert ~/.pulp/user-cert.pem"
 
+REPOS=("pulp" "pulp_deb" "pulp_docker" "pulp_openstack" "pulp_ostree" "pulp_puppet" "pulp_python" "pulp_rpm")
+SERVICES=("goferd" "httpd" "pulp_workers" "pulp_celerybeat" "pulp_resource_manager" "pulp_streamer")
+
 pstart() {
     _paction start
 }
+_pstart_help="Start all pulp-related services"
 
 pstop() {
     _paction stop
 }
+_pstop_help="Stop all pulp-related services"
 
 prestart() {
     _paction restart
 }
+_prestart_help="Restart all pulp-related services"
 
 pstatus() {
     _paction status
 }
+_pstatus_help="Report the status of all pulp-related services"
 
 preset() {
+    echo "Due to its similarity to 'prestart', 'preset' has been renamed to 'pclean'."
+    echo "Please use 'pclean' instead, and remember to update any script that might be calling it."
+    sleep 3
+    pclean
```

Comment:
That, and also lets you at least know that a message was printed, if not actually read it.